src/linter-jscs.js

```diff
@@ -105,7 +108,11 @@ export default class LinterJSCS {
             this.fixString(editor);
           }
         }
-      });
+      }));
+      currentEditor.add(editor.onDidDestroy(() => {
+        currentEditor.dispose();
```

Comment:
I just wasn't sure if `currentEditor` referred to the correct editor when it would eventually be destroyed, since the referenced commit referred back to their equivalent of the Map, instead of using it directly.